index.js

```diff
@@ -127,9 +127,34 @@ function toggleDetails(details: Element, open: boolean) {
   }
 }
 
+function loadIncludeFragment(event: Event) {
+  const details = event.currentTarget
+  if (!(details instanceof Element)) return
+  const dialog = details.querySelector('details-dialog')
+  if (!(dialog instanceof DetailsDialogElement)) return
+  const state = initialized.get(dialog)
+  if (!state) return
+
+  if (!dialog.src) return
+  if (state.loading || state.loaded) return
+
+  const loader: any = dialog.querySelector('include-fragment')
```

Comment:
We can remove the `any` type to make the file `flow strict` and use `setAttribute('src')` instead of the `src` property.